.storybook/preview.scss

```diff
@@ -1,9 +1,18 @@
-@import "~carbon-components/src/globals/scss/styles";
+$feature-flags: (
+	ui-shell: true,
+);
 
+.carbon {
+	@import "~carbon-components/src/globals/scss/styles";
+}
 
 body {
-	padding: 20px;
 	// for tooltips/popovers ... can be on any chld element before the popover
 	// but body is most conveient
 	position: relative;
+	display: flex;
+}
+
+#root {
```

Comment:
Not sure what this does.. but it removes the center alignment styles that is on the body tag

Reply:
That was fixing things before everything was centered